src/components/Navigation/Navigation.jsx

```diff
@@ -1,67 +1,67 @@
+// Import External Dependencies
 import React from 'react';
 import Banner from 'react-banner';
+import DocSearch from 'docsearch.js';
+
+// Import Utilities/Images
+import GitHubIcon from '../../styles/icons/github.svg';
```

Comment:
Is this dependency used? I can't spot it. I'm seeing we are using the class for the icon font instead.

Reply:
Good catch, I’ll remove it. Played around with it momentarily and the fixed font metrics instead.